OpenTok/OpenTok.cs

```diff
@@ -300,6 +300,10 @@ public Archive StartArchive(string sessionId, string name = "", bool hasVideo =
                 {
                     throw new OpenTokArgumentException("Could not set layout, stylesheet must be set if and only if type is custom");
                 }
+                else if(layout.ScreenShareType != null && layout.Type != LayoutType.bestFit)
+                {
+                    throw new OpenTokArgumentException($"Could not set screenShareLayout, when screenShareType is set, layout.Type must be bestFit, was {layout.ScreenShareType}");
```

Comment:
Removing a comma, the sentence flow is weird with it
```suggestion
                    throw new OpenTokArgumentException($"Could not set screenShareLayout, when screenShareType is set layout.Type must be bestFit, was {layout.ScreenShareType}");
```

Reply:
I changed the sentence structure a bit - though in a different way than you commented